### PR TITLE
applications: nrf_desktop: Document common HID device identifiers

### PR DESCRIPTION
**Pull request by MarekPieta**
Change introduces documentation for common HID device identifiers.

Jira: NCSDK-19101

### CODE REVIEW

File: applications/nrf_desktop/README.rst [hunk 1]
```diff
@@ -965,6 +965,23 @@ Apart from the device role, the application-specific Kconfigs specify a set of s
...
+
+HID device identifiers
+~~~~~~~~~~~~~~~~~~~~~~
+
+The nRF Desktop application defines the following common device identifiers:
+
+* Manufacturer (:ref:`CONFIG_DESKTOP_DEVICE_MANUFACTURER <config_desktop_app_options>`),
+* Vendor ID (:ref:`CONFIG_DESKTOP_DEVICE_VID <config_desktop_app_options>`),
+* Product name (:ref:`CONFIG_DESKTOP_DEVICE_PRODUCT <config_desktop_app_options>`),
+* Product ID (:ref:`CONFIG_DESKTOP_DEVICE_PID <config_desktop_app_options>`).
```
Comment (annwoj) [inline applications/nrf_desktop/README.rst:978]:
```suggestion
* Manufacturer (:ref:`CONFIG_DESKTOP_DEVICE_MANUFACTURER <config_desktop_app_options>`)
* Vendor ID (:ref:`CONFIG_DESKTOP_DEVICE_VID <config_desktop_app_options>`)
* Product name (:ref:`CONFIG_DESKTOP_DEVICE_PRODUCT <config_desktop_app_options>`)
* Product ID (:ref:`CONFIG_DESKTOP_DEVICE_PID <config_desktop_app_options>`)
```

File: applications/nrf_desktop/README.rst [hunk 2]
```diff
@@ -965,6 +965,23 @@ Apart from the device role, the application-specific Kconfigs specify a set of s
...
+
+These Kconfig options determine default values of device identifiers used for:
```
Comment (annwoj) [inline applications/nrf_desktop/README.rst:980]:
```suggestion
These Kconfig options determine the default values of device identifiers used for:
```

File: applications/nrf_desktop/README.rst [hunk 3]
```diff
@@ -965,6 +965,23 @@ Apart from the device role, the application-specific Kconfigs specify a set of s
...
+
+* :ref:`nrf_desktop_usb_state_identifiers`,
```
Comment (annwoj) [inline applications/nrf_desktop/README.rst:982]:
```suggestion
* :ref:`nrf_desktop_usb_state_identifiers`
```

File: applications/nrf_desktop/README.rst [hunk 4]
```diff
@@ -965,6 +965,23 @@ Apart from the device role, the application-specific Kconfigs specify a set of s
...
+* BLE GATT Device Information Service (:ref:`CONFIG_BT_DIS`) that is required for :ref:`nrf_desktop_bluetooth_guide_peripheral`.
```
Comment (annwoj) [inline applications/nrf_desktop/README.rst:983]:
```suggestion
* BLE GATT Device Information Service (:ref:`CONFIG_BT_DIS`) that is required for :ref:`nrf_desktop_bluetooth_guide_peripheral`
```

File: applications/nrf_desktop/doc/usb_state.rst [hunk 5]
```diff
@@ -26,15 +26,20 @@ Configuration
...
 
-When enabling the USB support for the device, set the following generic device options:
+Additionally, you can also configure the options described in the following sections.
+
+.. _nrf_desktop_usb_state_identifiers:
+
+USB device identifiers
+======================
+
+When USB support is enabled for the device, the following options are by default set to common :ref:`nrf_desktop_hid_device_identifiers` defined for the nRF Desktop application:
```
Comment (annwoj) [inline applications/nrf_desktop/doc/usb_state.rst:36]:
```suggestion
When the USB support is enabled for the device, the following options are by default set to common :ref:`nrf_desktop_hid_device_identifiers` defined for the nRF Desktop application:
```

File: applications/nrf_desktop/README.rst [hunk 6]
```diff
@@ -1566,6 +1583,7 @@ The HID over GATT profile specification requires Bluetooth Peripherals to define
 * HID Service - Handled in the :ref:`nrf_desktop_hids`.
 * Battery Service - Handled in the :ref:`nrf_desktop_bas`.
 * Device Information Service - Implemented in Zephyr and enabled with :kconfig:option:`CONFIG_BT_DIS`.
+  The device identifiers are by default configured according to the common :ref:`nrf_desktop_hid_device_identifiers`.
```
Comment (annwoj) [inline applications/nrf_desktop/README.rst:1586]:
```suggestion
  The device identifiers are configured according to the common :ref:`nrf_desktop_hid_device_identifiers` by default.
```

File: applications/nrf_desktop/README.rst (hunk 4 above)
Comment (divipillai) [inline applications/nrf_desktop/README.rst:983]:
```suggestion
* BLE GATT Device Information Service (:kconfig:option:`CONFIG_BT_DIS`) that is required for :ref:`nrf_desktop_bluetooth_guide_peripheral`.
```
This clears the doc build failure.

**Review by annwoj — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Review by mkapala-nordic — APPROVED** [review]

**Review by alstrzebonski — APPROVED** [review]

**Pull request merged**